promrules.yaml

```diff
@@ -57,27 +57,70 @@ additionalPrometheusRulesMap:
             expr: |
               avg_over_time(probe_http_duration_seconds[1m]) > 2
-            for: 5m
+            for: 2m
+            labels:
+              email_contact: pops@greenpeace.org
+              gitlab_project: planet4-documentation-and-issues
+              severity: medium
+              slack_channel: p4-slack-alert
+          - alert: PVCFullInFourDays
+            annotations:
+              description: >-
+                '{{ $labels.namespace }}/{{ $labels.persistentvolumeclaim }} is
+                expected to fill up within four days.   Currently {{ $value |
+                humanize }}% is available.
+              observability_url: >-
+                https://grafana.greenpeace.org/d/VrWrfr1Mz/kubernetes-persistent-volumes?orgId=1&refresh=10s&var-datasource=Thanos%20Global&var-name={{$externalLabels.name}}&var-namespace={{$labels.namespace}}&var-persistentvolumeclaim={{$labels.persistentvolumeclaim}}
+              runbook_url: >-
+                  https://docs.google.com/document/d/1YgbkEDgENyxJ67QZ5VeVTh-uZfv8UCKvDGv553CvqBQ/edit#bookmark=id.89vqtjr3cb8p
+              summary: '{{$labels.persistentvolumeclaim}} PV filling up'
+              gitlab_incident_markdown: |
+                  /label  ~\"Terraform Deployment\"\n PV Filling Up
+            expr: >-
+              predict_linear(kubelet_volume_stats_available_bytes[6h],
+              104 * 24 * 3600) < 0
+            for: 0m
+            labels:
+              email_contact: pops@greenpeace.org
+              gitlab_project: planet4-documentation-and-issues
+              severity: critical
+              slack_channel: p4-slack-alert
+          - alert: KubernetesVolumeOutOfDiskSpace
+            annotations:
+              description: >-
+                Kubernetes Volume out of disk space (instance
+                {{ $labels.persistentvolumeclaim }})
+              observability_url: >-
+                https://grafana.greenpeace.org/d/VrWrfr1Mz/kubernetes-persistent-volumes?orgId=1&refresh=10s&var-datasource=Thanos%20Global&var-name={{$externalLabels.name}}&var-namespace={{$labels.namespace}}&var-persistentvolumeclaim={{$labels.persistentvolumeclaim}}
+              runbook_url: >-
+                  https://docs.google.com/document/d/1YgbkEDgENyxJ67QZ5VeVTh-uZfv8UCKvDGv553CvqBQ/edit#bookmark=id.89vqtjr3cb8p
+              summary: '{{$labels.instance}} PV running out of disk space'
+              gitlab_incident_markdown: |
+                  /label  ~\"Terraform Deployment\"\n PV Filling Up
+            expr: >-
+              kubelet_volume_stats_available_bytes
+              / kubelet_volume_stats_capacity_bytes * 100 < 20
+            for: 0m
             labels:
-              email_contact: global-it-operations@greenpeace.org
+              email_contact: pops@greenpeace.org
               gitlab_project: planet4-documentation-and-issues
-              severity: warning
+              severity: critical
```

Comment:
A critical alert is basically one that requires someone to wake up and do a break/fix immediately. 

Depending on the service this _might_ require that but more likely we can detect that in another way better. (ie. ping checks for prod sites being critical, or the ELK service working being critical rather than PVC full/filling being critical)

We don't have pagers but if we did a critical alert is something someone would be paged for in theory.

let me know what you think